frontend/src/metabase/query_builder/VisualizationError.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { Component, PropTypes } from 'react';
+import { findDOMNode } from 'react-dom';
+
+import MetabaseSettings from "metabase/lib/settings";
+import VisualizationErrorMessage from './VisualizationErrorMessage';
+
+const EmailAdmin = () => {
+  const adminEmail = MetabaseSettings.adminEmail()
+  return adminEmail && (
+      <span className="QueryError-adminEmail">
+          <a className="no-decoration" href={`mailto:${adminEmail}`}>
+              {adminEmail}
+          </a>
+      </span>
+  )
+}
+
+class VisualizationError extends Component {
+  constructor (props) {
+      super(props)
+  }
+
+  showDetailError() {
+      if (this._detailErrorLink && this._detailErrorBody ) {
+          findDOMNode(this._detailErrorLink).style.display = "none";
```

Comment:
Yeah its pretty gross.  I'll add a ``// TODO: warcrime `` note so we can take a look at changing it or see if its trivial to change.